packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -111,31 +114,34 @@ export function getReceiptsMessageOut(receipts: TransactionResultReceipt[]) {
 }
 
 /** @hidden */
-const mergeAssets = (op1: Operation, op2: Operation) => {
+type Asset = {
+  assetId: string;
+  amount: BNInput;
+};
+
+export function mergeAssets(op1: Operation, op2: Operation): Asset[] {
```

Comment:
```suggestion
/** @hidden */
function mergeAssets(op1: Operation, op2: Operation): Asset[] {
```

Is this method to be used externally?